docs/4.4/api-reference.md

```diff
@@ -176,9 +428,58 @@ if err := client.DeleteRole(ctx, "auditor"); err != nil {
 
 ## Tokens
 
-[Tokens](admin-guide.md#adding-nodes-to-the-cluster) are used to add nodes to clusters, and add leaf clusters to [trusted clusters](admin-guide.md#trusted-clusters).
+Teleport is a "clustered" system, meaning it only allows access to hosts that had been previously granted cluster membership. To achieve this, a cluster has "join tokens" which can be shared to extend trust.
+
+A remote host can exchange one of these tokens with the cluster's auth server to receive signed certificates and become a trusted Teleport host (auth, node, or proxy server). Likewise, a remote Teleport cluster can exchange a token to become a leaf cluster in a [trusted cluster](admin-guide.md#trusted-clusters).
```

Comment:
You can technically join a cluster as an `app` server now too (and a `kubernetes` server at some point soon as well) - maybe we can leave this for a future edit though.

Reply:
Yeah I'd prefer to change it once it's in the code so I can pull some of it out.